cocotb/share/makefiles/Makefile.sim

```diff
@@ -131,6 +131,10 @@ PWD := $(shell pwd)
 
 include $(COCOTB_SHARE_DIR)/makefiles/simulators/Makefile.$(SIM_LOWERCASE)
 
+define check_for_results_file
+    @test -f $(COCOTB_RESULTS_FILE) || (echo "ERROR: $(COCOTB_RESULTS_FILE) does not exist!" >&2 && exit 1)
```

Comment:
```suggestion
    @test -f $(COCOTB_RESULTS_FILE) || (echo "ERROR: $(COCOTB_RESULTS_FILE) was not written by the simulation!" >&2 && exit 1)
```
Otherwise the user might think it was supposed to be there already.